### PR TITLE
transfersh: add page

### PR DESCRIPTION
**Pull request by marchersimon**
This client has more features, but I only documented those which actually worked.

### CODE REVIEW

File: pages/common/transfersh.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`transfersh {{path/to/file}}`
+
+- Upload a file showing a progress bar (requires Python package `requests_toolbelt`):
+
+`transfersh --progress {{path/to/file}}`
+
+- Upload a file using a different file name:
+
+`transfersh --name {{file_name}} {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/transfersh.md:16]:
```suggestion
`transfersh --name {{filename}} {{path/to/file}}`
```

File: pages/common/transfersh.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Upload a file to a custom transfer.sh server:
+
+`transfersh --servername {{upload.server.name}} {{path/to/file}}`
+
+- Upload all files in a directory recursively:
```
Comment (navarroaxel) [inline pages/common/transfersh.md:22]: from?
```suggestion
- Upload all files from a directory recursively:
```

Comment (marchersimon) [reply pages/common/transfersh.md:22]: Not sure. `From` sounds like you're moving the files.

Comment (sbrl) [reply pages/common/transfersh.md:22]: Either would work fine here.

**Review by blueskyson — APPROVED** [review]
Looks fine.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**